README.md

```diff
@@ -1,3 +1,20 @@
+# Doubl_N Games API
+
+https://doubl-n-games.herokuapp.com/api
+
+## Project Summary
+This project is a REST API for a database of board game reviews, made up of reviews, categories, comments, and users tables. It currently supports GET, POST, PATCH, and DELETE requests at a variety of endpoints. A 'table of contents' of available requests and endpoints can be seen at GET `/api` or at the link above. 
+
 ## Project Setup
 
-Please create a .env.development file and assign the development database to the `PGDATABASE` variable, and a .env.test file and assign the test database to the `PGDATABASE` variable to be able to connect to the development and test databases.
\ No newline at end of file
+### Cloning the project and installing dependencies
+1. Clone the project using `git clone https://github.com/doublN/nc-games.git`
```

Comment:
I'd add a step to add .env files, you could provide an example to make the setup order clearer 
You could expand the section below - eg: 
``` 
PGDATABASE=nc_games
```